images/openshift-enterprise-operator-sdk.yml

```
@@ -0,0 +1,29 @@
+container_yaml:
```

please add the following until we have a successful build through ART:

```suggestion
mode: wip
container_yaml:
```